app/views/users/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<% if !flash.now[:error].nil? %>
```

Comment:
You can dry this up by putting a single flash error checker in your application layout. That way, you wouldn't need to include this at the top of individual view files.